### PR TITLE
kibana 5.4.1

### PR DESCRIPTION
**Pull request by jasontedor**
This pull request bumps the version on the Kibana formula from version 5.4.0 to version 5.4.1.

### CODE REVIEW

**Comment by ilovezfs** [comment]
@jasontedor looks like a minor issue
```
Error: inreplace failed
/private/tmp/kibana-20170601-55631-120hiuk/tasks/build/notice.js:
  expected replacement of /linux-x64/ with "darwin-x64"
```

File: Formula/kibana.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def install
     end
 
     # do not build packages for other platforms
-    platforms = Set.new(["darwin-x64", "linux-x64", "linux-x86", "windows-x86"])
+    platforms = Set.new(["darwin-x64", "linux-x64", "winows-x64"])
```
Comment (ilovezfs) [inline Formula/kibana.rb]: what is a winows? :)

Comment (jasontedor) [inline Formula/kibana.rb]: A typo. 😞

Comment (ilovezfs) [inline Formula/kibana.rb]: Darn! I was hoping it was a kind of :bird:

**Comment by ilovezfs** [comment]
Shipped! Thanks for fixing it so quickly @jasontedor and for the upgrade.

**Comment by jasontedor** [comment]
Thanks much @ilovezfs.

**Pull request closed without merging**